### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by vinaysshenoy**

### CODE REVIEW

**Comment by msasikanth** [comment]
Since @Honey14 is on leave, assigning this to myself.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Next Release
+## On Demo
 ### Features
 - Refill prescriptions in edit medicine screen
 - [In Progress: 11 Nov 2020] Migrate app screens to use ViewBinding
```
Comment (msasikanth) [inline CHANGELOG.md:6]: Should we extract out these in-progress items into the next release section?

**Review by msasikanth — CHANGES_REQUESTED** [review]
Looks good. Left one comment PTAL

File: CHANGELOG.md (hunk 1 above)
Comment (vinaysshenoy) [reply CHANGELOG.md:6]: Ah yes, I'll do this.

**Review by msasikanth — APPROVED** [review]

**Pull request merged**